tripy/tripy/frontend/module/module.py

```diff
@@ -203,6 +203,15 @@ def find_module(module: Union[Module, List, Dict], sub_strs: List[str]):
                     module = operator.attrgetter(child_name)(module)
             return module
 
+        expected_keys = set(self.state_dict())
```

Comment:
Wouldn't this be a `Set[Tuple[str, Parameter]]`? Or does it only get the keys?

Reply:
It only get the keys, it's same as set(dictionary.keys())